stacks-core/create-source-binary/README.md

```diff
@@ -0,0 +1,33 @@
+# Create Source Binary action
+
+Builds a binary for the given architecture and uploads it to artifacts.
+
+## Documentation
+
+### Inputs
+
+| Input            | Description                   | Required | Default                  |
+| ---------------- | ----------------------------- | -------- | ------------------------ |
+| `arch`           | Binary's build architecture   | `true`   | null                     |
+
+## Usage
+
+```yaml
+name: Action
+on: push
+jobs:
+  build:
+    name: Job
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 10
+      matrix:
+        platform: ${{ fromJson(inputs.arch) }}
+    steps:
+      - name: Build Binary
+        id: build_binary
+        timeout-minutes: 30
+        uses: stacks-network/actions/stacks-core/create-source-binary@main
+        with:
+          arch: ${{ matrix.platform }}
```

Comment:
for the example, keep it simple. ex:
```suggestion
name: Action
on: push
jobs:
  build:
    name: Job
    runs-on: ubuntu-latest
    steps:
      - name: Build Binary
        id: build_binary
        uses: stacks-network/actions/stacks-core/create-source-binary@main
        with:
          arch: linux-glibc-x64
```

Reply:
https://github.com/stacks-network/actions/pull/19/files#r1476785672

adding the input tag, this example will need to be changed slightly:
```
...
        with:
	          arch: linux-glibc-x64
              tag: 0.0.0.1-rc1	          
```